src/tools/scripts/creationCoordinateur/index.js

```diff
@@ -112,7 +121,23 @@ execute(__filename, async ({ db, logger, exit, Sentry }) => {
               id: conum?.entity?.oid, nom: conum?.nom, prenom: conum?.prenom,
               nonAffichageCarto: coordoAnonyme.includes(String(idCoordinateur))
             };
-            if (mailleRegional?.length > 0) {
+            if (mailleCodePostaux?.length > 0) {
+              const listCodePostaux = mailleCodePostaux.split('/')?.map(e => e.trim());
+              let arrayConflictCodeCommune = [];
+              for (const codePostal of listCodePostaux) {
+                const urlAPI = `https://geo.api.gouv.fr/communes?codePostal=${codePostal}&format=geojson&geometry=centre`;
+                const { data } = await axios.get(urlAPI);
+                if (data?.features.length >= 2) {
+                  arrayConflictCodeCommune.push(codePostal);
+                }
+              }
+              if (arrayConflictCodeCommune.length >= 1) {
+                logger.warn(`Liste code Postaux ${arrayConflictCodeCommune} du coordinateur ${adresseCoordo} a plusieurs code commune !`);
+              }
+              listMaille = await getListCodeCommune(db)(listCodePostaux);
+              await addListSubordonnes(db)(idCoordinateur, listMaille, 'codeCommune');
```

Comment:
```suggestion
              await addSubordonnes(db)(idCoordinateur, listMaille, 'codeCommune');
```
Pas besoin du terme technique `List` puisque `Subordonnes` à un `s` donc il sait que c'est un tableau|liste|dictionnaire...